base.go

```diff
@@ -366,16 +366,30 @@ func getBuilt(o BaseLayerOpts, sfm StackerFiles) error {
 		// it was sourced from e.g. a tar file, in which case there's
 		// nothing we can do besides initialize an empty oci tag and
 		// generate the whole thing.
+
+		// Attempt to copy the base from o.Config.OCIDir layout in case it's already there
+		err = lib.ImageCopy(lib.ImageCopyOpts{
+			Src:  fmt.Sprintf("oci:%s:%s", o.Config.OCIDir, baseInputTag),
+			Dest: fmt.Sprintf("oci:%s:%s", o.Config.OCIDir, targetName),
+		})
+		if err == nil {
+			return nil
+		}
+		fmt.Printf("warning: %s", err)
```

Comment:
I'd really rather not print these errors if we expect them. I probably set a bad example by my "just give up" comment.

But in this particular case, I'm not sure what this code does. Isn't it always not there? I don't think this should ever succeed.